### ISSUE TITLE
Seems like forever...

### ISSUE THREAD

**Issue by kc3bbq** [xoreaxeaxeax/..cantor.dust..#2]
It's been a few years waiting and I see today that its still not shared. Will it ever be shared? I am currently working with Veles for my needs but there is so much more to cantor.dust from what I have seen that I am very interested in getting my hands on it.

**Comment by adde88** [comment]
Yeah, i really hope that this project was more than a publicity stunt....
Perhaps someone saw the potential and purchased it, who knows?
Any information at all would be great. I see soooo more much use in Cantor Dust compared to Veles 😄
